File: datasets/IR-Plag/convertai-original-java-rest-python/case-04/plagiarized/L6/01/L6.py
def print(mil):
    for i in range(10, 0, -1):
        print(f"{mil}\t\t{mil * 1.609}")
        mil += 1

if __name__ == "__main__":
    mil = 1
    print("Miles\t\tKilometers")
    print("-------------------------------")
    print(mil)

File: datasets/IR-Plag/intellij-original-java-rest-kotlin/case-04/plagiarized/L6/01/L6.kt
package IR

object L6 {
    fun print(mil: Int) {
        var mil = mil
        for (i in 10 downTo 1) {
            println(mil.toString() + "\t\t" + mil * 1.609)
            mil++
        }
    }

    @JvmStatic
    fun main(args: Array<String>) {
        val mil = 1
        println("Miles\t\tKilometers")

        println("-------------------------------")

        print(mil)
    }
}
